toolchain/check/convert.cpp

```diff
@@ -420,6 +420,15 @@ static auto ConvertStructToStruct(Context& context, SemIR::StructType src_type,
     return SemIR::NodeId::BuiltinError;
   }
 
+  // Prepare to look up fields in the source by index.
+  llvm::SmallDenseMap<StringId, int32_t> src_field_indexes;
```

Comment:
The plural is indices. 😛

Reply:
Our style guide says to use "indexes" (https://developers.google.com/style/word-list#index).